src/scaffoldmaker/meshtypes/meshtype_3d_lung1.py

```diff
@@ -780,3 +684,496 @@ def refineMesh(cls, meshrefinement, options):
         refineElementsCount = options['Refine number of elements']
         meshrefinement.refineAllElementsCubeStandard3d(refineElementsCount, refineElementsCount, refineElementsCount)
 
+def getLungNodes(lungSide, cache, coordinates, generateParameters, nodes, nodetemplate, nodeFieldParameters,
+                 lElementsCount1, lElementsCount2, lElementsCount3,
+                 uElementsCount1, uElementsCount2, uElementsCount3,
+                 lowerNodeIds, upperNodeIds, nodeIndex, nodeIdentifier):
+    """
+    :parameter:
```

Comment:
:param lowerNodeIds:  Description, say filled by this function, including indexing by [lElementsCount3 + 1][lElementsCount2 + 1][lElementsCount1 + 1]
:param upperNodeIds: As above